app/src/main/res/layout/activity_login.xml

```diff
@@ -1,17 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_marginHorizontal="@dimen/activity_horizontal_margin"
+    android:layout_marginVertical="@dimen/activity_vertical_margin">
 
-    <Button
-        android:id="@+id/button"
+    <TextView
+        android:id="@+id/textView"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="Sign in" />
+        android:textSize="26sp"
+        android:textColor="#000000"
+        android:textStyle="bold"
+        android:text="You are not signed in" />
```

Comment:
Do not hardcode the string. And since you are setting it programmatically, you don't need to set the text here. If you want to see the design on preview window, you can use tools:text instead